lib/pure/terminal.nim

```diff
@@ -462,15 +509,35 @@ elif not defined(windows):
     result = stdin.readChar()
     discard fd.tcsetattr(TCSADRAIN, addr oldMode)
 
+# Deprecated procs assuming output to stdout:
+template setCursorPos*(x, y: int) {.deprecated.} = setCursorPos(stdout, x, y)
```

Comment:
I am not sure if I am following you. All these procs assuming `stdout` are "deprecated" but not removed (still provided), the code will still compile but will emit warnings - just to encourage programmers to specify explicitly where is the terminal. So isn't that an optional "deprecated"?

> Believe it or not, not everybody cares about 'stderr'. ;-)

Hmmz... Believe me or not, all tools I know and compilers I know emit (colored) diagnostics to `stderr`, where apps meant to be used in console only such as `htop` or `mc` use `stdout`. So definitely there are ppl that care. So one cannot make valid assumption about where is terminal without knowing what the app is actually doing.

As I said I can leave `setCursorPos` not deprecated, but then it will likely cause that someone using non-`stdout` terminal will use that version, effectively emitting terminal control codes to wrong handle. So it is your pick.